upgrade/satellite.py

```diff
@@ -74,189 +71,150 @@ def satellite6_setup(os_version):
     return sat_host
 
 
-def satellite6_upgrade():
-    """Upgrades satellite from old version to latest version.
+def satellite6_upgrade(upgrade_type=None):
```

Comment:
Docstring is missing for upgrade_type parameter.

Reply:
Incorporated.